io_context/include/io_context/common.hpp

```diff
@@ -17,6 +17,7 @@
 #ifndef IO_CONTEXT__COMMON_HPP_
 #define IO_CONTEXT__COMMON_HPP_
 
-#include <asio.hpp>
+//#include <asio.hpp>
```

Comment:
What's the rationale for using Boost.Asio and not Asio? Using Boost.Asio brings in a lot of dependencies, whereas Asio only requires a C++11/17 compiler

Reply:
Addressed [here](https://github.com/ros-drivers/transport_drivers/pull/84/commits/17df5f06ba3cf8f822d9eecc47e820c1dd5dd188)